style_editor.html

```diff
@@ -7785,6 +7802,26 @@
   blade.addEffect(STATE_ON ? EFFECT_IGNITION : EFFECT_RETRACTION, Math.random() * 0.7 + 0.2);
 }
 
+function OnMoreChange() {
+  var dropdown = document.getElementById("MORE");
+  var selectedValue = dropdown.value;
+  switch(selectedValue) {
```

Comment:
I think there is a way to look up the effect number by name in the enum class, I'll have to check what it is though.